### ISSUE TITLE
Form: Screen reader does not announce the label of form items

### ISSUE THREAD

**Issue by gurkirpalgill** [SAP/ui5-webcomponents-react#4144]
**Describe the bug**
NVDA screen reader does not announce the label when tabbing to the form items.

**To Reproduce**

1. Steps to reproduce the behavior:
2. Open NVDA Screen Reader on Windows machine.
3. Open this form link: [Form](https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-form--default-story)
4. Tab into any of the form elements. Label value is not announced.

**Expected behavior**

- A clear and concise description of what you expected to happen.
- Try NVDA Screen Reader on form items on this [SAP UI5 Form Sample](https://sapui5.hana.ondemand.com/#/entity/sap.ui.webc.main.Input/sample/sap.ui.webc.main.sample.Input)
- As you can see label values are announced.

**Screenshots**
![Screenshot 2023-02-13 at 4 03 45 PM](https://user-images.githubusercontent.com/65705637/218604135-f51ae615-b846-4b7d-b43e-d144052fcf1f.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: ^1.9.1
`@ui5/webcomponents-react` version: ^1.5.0
Operating System: Windows
Browser: Chrome
